dprof.js

```diff
@@ -147,7 +147,17 @@ asyncHook.enable();
 // Print result
 //
 
-process.on('exit', function () {
+if (process.argv.indexOf('--raw-sigint') === -1 ||
```

Comment:
perhaps call it `--dprof-sigint`. The `raw` part doesn't really make sense to me.